entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -9,99 +9,218 @@ uid: core/get-started/aspnetcore/new-db
 
-* Open **Startup.cs**
-* Add the following `using` statements:
+# [Visual Studio](#tab/visual-studio)
 
-  [!code-csharp[Main](../../../../samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb/Startup.cs#AddedUsings)]
+* In **Startup.cs** add the following `using` statements:
 
-Call the `AddDbContext` method to register the context as a service.
+  [!code-csharp[Main](../../../../samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb/Startup.cs#AddedUsings)]
 
 * Add the following highlighted code to the `ConfigureServices` method:
 
   [!code-csharp[Main](../../../../samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb/Startup.cs?name=ConfigureServices&highlight=13-14)]
 
-Note: A real app would generally put the connection string in a configuration file or environment variable. For the sake of simplicity, this tutorial defines it in code. See [Connection Strings](../../miscellaneous/connection-strings.md) for more information.
+# [.NET Core CLI](#tab/netcore-cli)
+
+* In **Startup.cs** add the following `using` statements:
+
+  [!code-csharp[Main](../../../../samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb.Sqlite/Startup.cs#AddedUsings)]
+
+* Add the following highlighted code to the `ConfigureServices` method:
+
+  [!code-csharp[Main](../../../../samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb.Sqlite/Startup.cs?name=ConfigureServices&highlight=13-14)]
+
+
+---
+
+A production app would typically put the connection string in a configuration file or environment variable. For the sake of simplicity, this tutorial defines it in code. See [Connection Strings](../../miscellaneous/connection-strings.md) for more information.
 
 ## Create the database
 
-Once you have a model, you can use [migrations](https://docs.microsoft.com/aspnet/core/data/ef-mvc/migrations#introduction-to-migrations) to create a database.
+The following steps use [migrations](https://docs.microsoft.com/aspnet/core/data/ef-mvc/migrations#introduction-to-migrations) to create a database.
+
+# [Visual Studio](#tab/visual-studio)
 
 * **Tools > NuGet Package Manager > Package Manager Console**
-* Run `Add-Migration InitialCreate` to scaffold a migration to create the initial set of tables for your model. If you receive an error stating `The term 'add-migration' is not recognized as the name of a cmdlet`, close and reopen Visual Studio.
+* Run `Add-Migration InitialCreate` to scaffold a migration to create the initial set of tables for the model. If you receive an error stating `The term 'add-migration' is not recognized as the name of a cmdlet`, close and reopen Visual Studio.
 * Run `Update-Database` to apply the new migration to the database. This command creates the database before applying migrations.
 
+# [.NET Core CLI](#tab/netcore-cli)
+
+* Run `dotnet ef migrations add InitialCreate` to scaffold a migration to create the initial set of tables for the model.
```

Comment:
My pref is

Run the following commands

```cli
dotnet ef migrations add InitialCreate
dotnet ef database update
```
![image](https://user-images.githubusercontent.com/3605364/45059603-651c6780-b051-11e8-88dc-be335ce71e7b.png)

The xxx command scaffolds ...
The update command applies

That way your not mixing command and what they do. It's also much fast to click the copy code button and paste. Makes it much faster to test the tutorial.

You could take the same approach on the VS instructions. When updating a tutorial, I typically review the VS instructions but actually do the CLI instructions.